mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -407,7 +411,7 @@ private void handleCropResult(@NonNull Intent result) {
     public void clickProfilePicFromCamera() {
 
         if (Build.VERSION.SDK_INT >= 23 && ContextCompat.checkSelfPermission(this,
-                Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
+                                                                             Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
No need for indentation

Reply:
Thanks for reviewing and clearing. Should I change and make a PR again?